llvm/lib/DebugInfo/LogicalView/Core/LVType.cpp

```diff
@@ -292,7 +292,10 @@ void LVType::print(raw_ostream &OS, bool Full) const {
 }
 
 void LVType::printExtra(raw_ostream &OS, bool Full) const {
-  OS << formattedKind(kind()) << " " << formattedName(getName()) << "\n";
+  OS << formattedKind(kind()) << " " << formattedName(getName());
+  if (uint32_t Size = getStorageSizeInBytes())
+    OS << " [Size = " << Size << "]";
+  OS << "\n";
```

Comment:
What about the `printExtra` for `LVScope`?

Reply:
Ah, right - this is fixed now :+1:!  PTAL.